### DISCUSSION TITLE
API client included?

### DISCUSSION THREAD

**Discussion by lnaie** [PrometheusClientNet/Prometheus.Client#221] [Q&A] [answered]
Hello,

I was looking around for a solution and found this library.
Is there a typed API client abstraction to make requests to: https://prometheus.io/docs/prometheus/latest/querying/api/?

Thanks

**Comment by phnx47** [comment, marked as answer]
@lnaie Hello,

No API client here.

**Reply by govorovvs** [reply]
Hm, Why is the library called `Prometheus.Client` in this case?

**Reply by phnx47** [reply]
@govorovvs 

Prometheus - because lib for https://prometheus.io
Client - because client lib https://prometheus.io/docs/instrumenting/writing_clientlibs/
